bagel/cli.py

```diff
@@ -440,33 +446,28 @@ def derivatives(
     ):
         existing_subject = existing_subs_dict.get(subject)
 
-        # Get existing imaging sessions for the subject
-        # Note: This dictionary can be empty if only bagel pheno was run
-        existing_sessions_dict = dutil.get_imaging_session_instances(
+        # Note: Dictionary of existing imaging sessions can be empty if only bagel pheno was run
+        existing_sessions_dict = get_imaging_session_instances(
             existing_subject
         )
 
-        for session_name, sub_ses_proc_df in sub_proc_df.groupby(
+        for session, sub_ses_proc_df in sub_proc_df.groupby(
             PROC_STATUS_COLS["session"]
         ):
-            # Create pipeline objects for the subject-session
             completed_pipelines = dutil.create_completed_pipelines(
                 sub_ses_proc_df
             )
 
             if not completed_pipelines:
                 continue
-            if session_name in existing_sessions_dict:
-                existing_img_session = existing_sessions_dict.get(session_name)
+
+            session_label = CUSTOM_SESSION_ID if session == "" else session
```

Comment:
Ah, I see. I feel like this is more clearly done by reassigning `session_name` or `session_label`, rather than calling it `session` first.